server/build.gradle

```diff
@@ -134,11 +135,15 @@ dependencies {
     // tests use the locally compiled version of server
     exclude group: 'org.elasticsearch', module: 'server'
   }
+  internalClusterTestCompile(project(":test:framework")) {
```

Comment:
should the plugin setup the TestCompile dependency on `:test:framework` ? (admittedly, I don't know how to handle the exclude here...but i think all tests will need that dependency)

Reply:
Possibly, but I'd like to consolidate that after we have converted all the xpack projects to using this plugin. For now, I want to stay simple as possible and only make it more complicated as we see universal commonalities.